JetAnalyzers/BuildFile.xml

```diff
@@ -23,9 +23,9 @@
 <use   name="rootcore"/>
 <use   name="rootMinuit"/>
 <use   name="rootspectrum"/>
-<flags   EDM_PLUGIN="1"/>
 <!--
+<flags   EDM_PLUGIN="1"/>
+-->
```

Comment:
I used:
<export>
  <lib   name="1"/>
</export>

instead of:
<flags   EDM_PLUGIN="1"/>

to make the src/ compile.